src/Nelmio/Alice/Persister/Doctrine.php

```diff
@@ -55,4 +60,21 @@ public function find($class, $id)
 
         return $entity;
     }
+
+    private function getPersistableClasses()
```

Comment:
it looks like this operation could be cached somehow and not being called at each persist.